examples/hybrid/src/main.rs

```diff
@@ -0,0 +1,53 @@
+#![feature(let_chains)]
+
+mod grpc;
+mod web;
+
+mod hybrid;
+
+use std::{convert::Infallible, net::SocketAddr};
+
+use axum::response::Response;
+use hyper::{server::conn::AddrStream, service::make_service_fn};
+use tower::ServiceExt;
+use tracing::info;
+use tracing_subscriber::FmtSubscriber;
+
+pub type BoxError = Box<dyn std::error::Error + Send + Sync>;
+pub type Result<T> = std::result::Result<T, BoxError>;
```

Comment:
`BoxError` also exists in axum as `axum::BoxError` and I don't think we should have a `Result` alias in the example. It wont be used in many places and will just confuse people.